StarWars.Core/Services/ObservableCollectionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace StarWars.Core.Services;
+
+public static class ObservableCollectionExtensions
+{
+    public static void ReplaceWith<T>(this ObservableCollection<T> collection, IEnumerable<T> items)
+    {
+        collection.Clear();
+        foreach (var item in items)
```

Comment:
oh, no, you should not do this, eack call of collection.Add pefrorm collection redraw

Reply:
please use MvxObservableCollection, there is correct realization of this from the box